lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -728,6 +740,8 @@ def test_gradebook(self):
         )
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
+        # Max number of student per page is one.  Patched setting MAX_STUDENTS_PER_PAGE_GRADE_BOOK = 1
+        self.assertEqual(len(response.mako_context['students']), 1)  # pylint: disable=no-member
```

Comment:
Are you certain you need this pylint: disable? We merged an upgrade to our linting system recently (maybe yesterday?) which should lessen the need for these disables.